test/test-ipld-dag-cbor.js

```diff
@@ -0,0 +1,205 @@
+/* eslint-env mocha */
+'use strict'
```

Comment:
Similar, but not the same, cbor is way more flexible and needs even more throughout tests.